app/lib/actions/katello/content_view/incremental_updates.rb

```diff
@@ -67,7 +67,65 @@ def run
         end
 
         def humanized_name
-          _("Incremental Update")
+          total_count = total_counts(input)
+          _("Incremental Update of %s Content View Version(s) " % total_count[:content_view_count]) + content_output(total_count)
```

Comment:
Translations will not be able to properly translate this string, since some of the text is being derived within 'content_output'.